pages/linux/datamash.md

```diff
@@ -0,0 +1,19 @@
+# datamash
+
+> GNU datamash performs basic numeric, textual and statistical operations on input textual data files.
```

Comment:
There's no need to mention that name of the tool again in the heading. We can just say - `Tool to perform basic ..`